src/main/mw.c

```diff
@@ -523,8 +523,13 @@ void taskMainPidLoop(void)
     cycleTime = getTaskDeltaTime(TASK_SELF);
     dT = (float)cycleTime * 0.000001f;
 
-    imuUpdateAccelerometer();
-    imuUpdateGyroAndAttitude();
+    /*
+     * Acc update is run in sync with PID loop only when RACE mode is not enabled
+     */
+    if (!feature(FEATURE_RACE) && sensors(SENSOR_ACC)) {
```

Comment:
I think it's safe to run ACC and ATTI unsynched always as long as ACC has high enough sample rate